### PR TITLE
GLSK on Xnodes

### PR DESCRIPTION
**Pull request by baptsegu**
**Please check if the PR fulfills these requirements** *(please use `'[x]'` to check the checkboxes, or submit the PR and then click the checkboxes)*
- [X] The commit message follows our guidelines
- [X] Tests for the changes have been added (for bug fixes / features)

Addition in distribution package of utility network import post-precessor scripts, that are used in CORE CC process.

Small adjustement of the readme to mention those files.

### CODE REVIEW

File: distribution/src/main/resources/core-cc-ucte-postProcessor.groovy [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+
+createGeneratorOnXnode("XLI_OB1B")
+createGeneratorOnXnode("XLI_OB1A")
+
+void createGeneratorOnXnode(String xnodeId) {
+    Optional<DanglingLine> danglingLine = network.danglingLineStream.filter({dl -> dl.getUcteXnodeCode().equals(xnodeId)}).findAny()
+
+    if (!danglingLine.present) {
+
+        // System.out.println("Xnode " + xnodeId + " not found in network")
```
Comment (pet-mit) [inline distribution/src/main/resources/core-cc-ucte-postProcessor.groovy:62]: can you remove this line or replace it with a comment ?

File: distribution/src/main/resources/core-cc-ucte-postProcessor.groovy [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+        .setId(xnodeId + "_generator")
+        .setMaxP(9999)
+        .setMinP(0)
+        .setTargetP(0)
+        .setTargetQ(0)
+        .setTargetV(xNodeBus.getVoltageLevel().getNominalV())
+        .setVoltageRegulatorOn(false)
+        .add()
+
+        // System.out.println("Generator added in network for Xnode " + xnodeId)
```
Comment (pet-mit) [inline distribution/src/main/resources/core-cc-ucte-postProcessor.groovy:77]: same

File: distribution/src/main/resources/create-generators-on-alegro-xnodes.groovy [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+
+createGeneratorOnXnode("XLI_OB1B")
+createGeneratorOnXnode("XLI_OB1A")
+
+void createGeneratorOnXnode(String xnodeId) {
+    Optional<DanglingLine> danglingLine = network.danglingLineStream.filter({dl -> dl.getUcteXnodeCode().equals(xnodeId)}).findAny()
+
+    if (!danglingLine.present) {
+        
+        // System.out.println("Xnode " + xnodeId + " not found in network")
```
Comment (pet-mit) [inline distribution/src/main/resources/create-generators-on-alegro-xnodes.groovy:31]: same

File: distribution/src/main/resources/create-generators-on-alegro-xnodes.groovy [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+        .setId(xnodeId + "_generator")
+        .setMaxP(9999)
+        .setMinP(0)
+        .setTargetP(0)
+        .setTargetQ(0)
+        .setTargetV(xNodeBus.getVoltageLevel().getNominalV())
+        .setVoltageRegulatorOn(false)
+        .add()
+        
+        // System.out.println("Generator added in network for Xnode " + xnodeId)
```
Comment (pet-mit) [inline distribution/src/main/resources/create-generators-on-alegro-xnodes.groovy:46]: same

**Review by pet-mit — CHANGES_REQUESTED** [review]

File: distribution/src/main/resources/core-cc-ucte-postProcessor.groovy (hunk 1 above)
Comment (baptsegu) [reply distribution/src/main/resources/core-cc-ucte-postProcessor.groovy]: done, replaced by a comment

File: distribution/src/main/resources/core-cc-ucte-postProcessor.groovy (hunk 2 above)
Comment (baptsegu) [reply distribution/src/main/resources/core-cc-ucte-postProcessor.groovy]: done, replaced by a comment

File: distribution/src/main/resources/create-generators-on-alegro-xnodes.groovy (hunk 4 above)
Comment (baptsegu) [reply distribution/src/main/resources/create-generators-on-alegro-xnodes.groovy]: done, replaced by a comment

File: distribution/src/main/resources/create-generators-on-alegro-xnodes.groovy (hunk 3 above)
Comment (baptsegu) [reply distribution/src/main/resources/create-generators-on-alegro-xnodes.groovy]: done, replaced by a comment

**Review by pet-mit — APPROVED** [review]

**Pull request merged**